uportal-war/src/main/resources/layout/theme/respondr/navigation.xsl

```diff
@@ -55,6 +55,7 @@
 <xsl:param name="CONTEXT" select="'header'"/>
 <xsl:param name="subscriptionsSupported">true</xsl:param>
 <xsl:param name="USE_FLYOUT_MENUS" select="'false'" /> <!-- Sets the use of flyout menus.  Values are 'true' or 'false'. -->
+<xsl:param name="USE_OFFCANVAS_MENU" select="'true'" /> <!-- Sets the use of off-canvas menu.  Values are 'true' or 'false'. -->
```

Comment:
Yes it was in Universality but wasn't working in Respondr : you were opening the submenus on hover event and with fluid. Now you are opening them on clic event and with bootstrap, that's better for accessibility.
I think they have their utility because on mobile you can see, without firing all the request for a new tab, the content of a tab and can choose directly the portlet you want without scrolling down.